0.3L support for wordpress 6.2 & PHP 8

dalacost opened the pull request:
Adding support for wordpress 6.2 & PHP 8

dalacost committed:
support new Thingiverse web

Commit:
0.3L support for wordpress 6.2 & PHP 8

0.3L support for wordpress 6.2 & PHP 8

martymcguire commented:
Oh, wow, haha! Thanks @dalacost !

I haven't touched this in years and I don't look forward to maintaining it further. Would you be interested in maintaining your own version of it?

If you're up for it, feel free to take it along with your modifications to a new wordpress.org extension, and i'll update this one (http://wordpress.org/extend/plugins/thingiverse-embed/) to point there before I archive this repo.

Thanks again!!

dalacost commented:
Hi Marty, I been using this plugin for years on my website, I spect
continue using it for a few years more, so I could continue with the
support, no problem.

Commit:
new embed routine & new style

new routine to get thing information from thingiverse.com,  new template for embed thing.

Commit:
clear

Commit:
new widget style & options

Commit:
Delete thingiverse_logo.png

dalacost commented:
I started a new fork for this project here https://github.com/dalacost/wp-thingiverse-press

martymcguire commented:
Nice! Thanks, and congratulations!

Are you up for creating and maintaining a https://wordpress.org/plugins/ entry as well? I've got this [old one that I closed a couple of years ago](https://wordpress.org/plugins/thingiverse-embed/). It'd be great to be able to point folks at a working one they can install directly from the WP admin.

Let me know what you think and I'll archive this repository, pointing folks to your preferred location either on GitHub or wp.org/plugins
